Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	1	create_and_make_expr_table
	1	find_best_tree
	1	write_dict_file
	4

rule find_best_tree:
    input: tmp/OneBig.var.aln
    output: /net/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/results/Paeru.nwk
    jobid: 1

Unable to set utime on symlink /net/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/results/Paeru.nwk. Your Python build does not support it.
Finished job 1.
1 of 4 steps (25%) done

rule write_dict_file:
    input: tmp/CH2500/sorted.rmdup.rpg, tmp/CH2502/sorted.rmdup.rpg, tmp/CH2522/sorted.rmdup.rpg, tmp/F1659/sorted.rmdup.rpg
    output: tmp/rpg_dict
    jobid: 5

Finished job 5.
2 of 4 steps (50%) done

rule create_and_make_expr_table:
    input: tmp/rpg_dict
    output: /net/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/results/expr.tab
    jobid: 2

    Error in rule create_and_make_expr_table:
        jobid: 2
        output: /net/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/results/expr.tab

RuleException:
CalledProcessError in line 100 of /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/CREATE_EXPR_TABLE.smk:
Command ' set -euo pipefail;  Rscript /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/.snakemake.4xprg881.collect_rpg_data.R ' returned non-zero exit status 127.
  File "/net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/CREATE_EXPR_TABLE.smk", line 100, in __rule_create_and_make_expr_table
  File "/home/thkuo/miniconda3/envs/phypal/lib/python3.6/concurrent/futures/thread.py", line 55, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /net/sgi/metagenomics/data/from_moni/old.tzuhao/seq2geno/dev_versions/v3/.snakemake/log/2018-07-11T135229.471835.snakemake.log
